sonic_data_client/db_client.go

```diff
@@ -300,37 +300,59 @@ func (c *DbClient) PollRun(q *queue.PriorityQueue, poll chan struct{}, w *sync.W
 	c.q = q
 	c.channel = poll
 
+	prevUpdates := make(map[string]bool)
```

Comment:
>prevUpdates [](http://example.com/codeflow?start=1&length=11)

This map will always increase size in lifetime of telemetry process. Why you need it?